cloudinit/sources/DataSourceOracle.py

```diff
@@ -136,6 +143,10 @@ def __init__(self, sys_cfg, *args, **kwargs):
         )
         self._network_config_source = KlibcOracleNetworkConfigSource()
 
+        url_params = self.get_url_params()
+        self.url_max_wait = url_params.max_wait_seconds
+        self.url_timeout = url_params.timeout_seconds
```

Comment:
need an _unpickle FTW here. Otherwise upgrade path will hit us from cached ds to new DS